commit 62cc581402193eaa6bad8eeb8eb13efef6a7d6b8
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2019-05-10T19:05:18+00:00 (5 months ago)
    
    TRITON-1652 net-agent should create NICs using check_owner=false

diff --git a/lib/nic-fsm.js b/lib/nic-fsm.js
index 984625a..dd4c1bc 100644
--- a/lib/nic-fsm.js
+++ b/lib/nic-fsm.js
@@ -309,9 +309,11 @@ NicFSM.prototype.state_create = function (S) {
         S.gotoState('waiting');
     }
 
-    self.log.info({ nic: self.local }, 'Syncing local NIC to NAPI');
+    var remcreate = Object.assign({ check_owner: false }, self.local);
 
-    self.app.napi.createNic(self.mac, self.local, S.callback(afterCreate));
+    self.log.info({ nic: remcreate }, 'Syncing local NIC to NAPI');
+
+    self.app.napi.createNic(self.mac, remcreate, S.callback(afterCreate));
 };
 
 NicFSM.prototype.state_update = function (S) {
